extensions/GlobalMessages/GlobalMessagesA.i18n.php

```diff
@@ -30,16 +30,16 @@
 
 :\'\'$2\'\'
 
-* Start of block: $8
-* Expiry of block: $6
-* Intended blockee: $7
-
-You may contact $1 or one of the other [[{{MediaWiki:Grouppage-sysop}}|administrators]] to discuss the block.
+* Start of block: $7
+* Expiry of block: $5
+* Intended blockee: $6
 
 Note that you may not use the "e-mail this user" feature unless you have a valid e-mail address registered in your [[Special:Preferences|user preferences]] and you have not been blocked from using it.
```

Comment:
We could remove this line because we've removed the email user feature a few years ago